package.json

```diff
@@ -97,8 +97,10 @@
     "wdio": "tt-wdio --config node_modules/terra-toolkit/config/wdio/wdio.conf.js --locales=['en','de']"
   },
   "dependencies": {
+    "@testing-library/react": "^10.0.4",
```

Comment:
Should move to devDependencies I reckon?

Reply:
oh yes...

Reply:
fixed: https://github.com/cerner/terra-application/pull/45/commits/b70e1ed81c38d6c315d28cfd80420d69f36f28c7